.travis.yml

```diff
@@ -18,7 +16,6 @@ matrix:
       otp_release: 20.2
     - elixir: 1.4.5
       otp_release: 19.3
-      env: COVERALLS=true CREDO=true
```

Comment:
Removing this line disables credo. It should be put into one of the matrix builds, probably the latest version.